sklearn/preprocessing/_data.py

```diff
@@ -646,15 +646,18 @@ class StandardScaler(_OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
 
     Standardization of a dataset is a common requirement for many
     machine learning estimators: they might behave badly if the
-    individual features do not more or less look like standard normally
```

Comment:
> This is my first pull request for this project. I changed part of the description because I noticed it mentioned standardization as a means to make the data look more Gaussian or like the standard normal distribution.

Here, it is not what the sentence says. The sentence says that estimators might behave badly if the assumption of features being normally distributed is broken. This is different from saying that the standardization is making the features follow a Normal distribution.

So the description seems completely fine to me here. I would revert the change.